samples/python/lineiterator.py

```diff
@@ -0,0 +1,28 @@
+import cv2 as cv
+
+img = cv.imread(cv.samples.findFile('butterfly.jpg'))
+pt1 = (30, 40)
+pt2 = (200, 90)
+h = cv.LineIterator_create(pt1, pt2)
```

Comment:
prefer to use `.create` notation instead of `_create`
```suggestion
h = cv.LineIterator.create(pt1, pt2)
```